tasks/selinux.yml

```diff
@@ -14,6 +14,16 @@
     state: present
   when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
 
+- name: "Authorizing {{ ssh_server_ports }} ports for selinux"
```

Comment:
Can you change this to "Authorize {{ ssh_server_ports }} ports for selinux", so the tense stays the same?